api/src/passage/migrations/0014_zonezwaarverkeer_zwaarverkeerhelpertable.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 2.2.20 on 2021-08-19 12:57
+
+import django.contrib.gis.db.models.fields
+from django.contrib.postgres.operations import CreateExtension
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('passage', '0013_remove_passage_privacy_check'),
+    ]
+
+    operations = [
+        CreateExtension('postgis'),
+        migrations.CreateModel(
+            name='ZoneZwaarVerkeer',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('passage_at_timestamp', models.IntegerField()),
+                ('passage_at_date', models.DateField()),
+                ('passage_at_year', models.IntegerField()),
+                ('passage_at_month', models.IntegerField()),
+                ('passage_at_day', models.IntegerField()),
+                ('passage_at_week', models.IntegerField()),
+                ('passage_at_day_of_week', models.IntegerField()),
+                ('passage_at_hour', models.IntegerField()),
```

Comment:
Waarom gebruik je voor de datetime allemaal aparte velden? Dat kan toch ook gewoon in één veld?

Reply:
Dit is een op een wat ze vanuit de klant willen. De verklaring; ze gebruiken dit in Tableau en Tableau kan dit niet zelf (waarschijnlijk).